paywall/src/unlock.js/startup.ts

```diff
@@ -1,38 +1,81 @@
-import { makeIframe, addIframeToDocument } from './iframeManager'
-import setupPostOffices, { normalizeConfig } from './setupPostOffices'
-import { UnlockWindow } from '../windowTypes'
+import { UnlockWindowNoProtocolYet } from '../windowTypes'
+import IframeHandler from './IframeHandler'
+import Wallet from './Wallet'
+import MainWindowHandler from './MainWindowHandler'
+import CheckoutUIHandler from './CheckoutUIHandler'
+import StartupConstants from './startupTypes'
 
-export default function startup(window: UnlockWindow) {
-  // Get the config
-  const normalizedConfig = normalizeConfig(window.unlockProtocolConfig)
+/**
+ * convert all of the lock addresses to lower-case so they are normalized across the app
+ */
+export function normalizeConfig(unlockConfig: any) {
```

Comment:
I think I have seen that code somewhere else? If that's the case do you mind adding a comment `TODO: DRY-up with ...`?

Reply:
it's used on line 42. Where you had seen it before was in `setupPostOffices.ts`, which is going to be deleted in the PR following this one as obsolete, so it's a straight move-for-refactor